presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/SchemaDaoUtil.java

```diff
@@ -57,6 +57,14 @@ private static void createTables(SchemaDao dao)
         dao.createTableDeletedShards();
         dao.createTableBuckets();
         dao.createTableShardOrganizerJobs();
+
+        try {
```

Comment:
It's better to check if those columns exist instead of using `try`.

Reply:
Currently I have tried two method to get the columns 
1) select column_name from information_schema.columns where table_name='tablename' and column_name='columnname'
the result is NULL
I suspect that we don't have the access to information_schema.

2) SHOW COLUMNS FROM `table` LIKE 'fieldname'
It seems not easy to achieve this function through jdbi framework.

But it can be written in JDBC style:
ResultSetMetaData rsmd = resultSet.getMetaData();
Which can make our SchemaDaoUtil ugly

Based on these, I decided to keep the ``try catch`` to make ``SchemaDaoUtil`` more consistent from code perspective